packages/svelte/src/internal/client/runtime.js

```diff
@@ -51,7 +52,7 @@ let current_queued_effects = [];
 /** @type {Array<() => void>} */
 let current_queued_tasks = [];
 /** @type {Array<() => void>} */
-let current_queued_microtasks = [];
+let current_queued_frames = [];
```

Comment:
Would avoid the plural `frames` here as that sounds a bit misleading (everything happens on the next frame, there are not multiple consecutive frames involved).

Maybe make it closer to the function, something like `next_frame_queue` or if you want to retain the prefix maybe `current_raf_tasks`.